tests/Unit/Training/WaitingListAccountTest.php

```diff
@@ -213,4 +213,19 @@ public function itCanHaveNotesAdded()
 
         $this->assertEquals('This is a note', $waitingListAccount->fresh()->notes);
     }
+
+    /** @test */
+    public function itCanHaveNotesModified()
+    {
+        $account = factory(Account::class)->create();
```

Comment:
I'm not sure how up to date the epic is with master, but in line with the testing housekeeping I did, `account` should become `user`, and if you are just creating a bog standard user you should use `$this->user` (https://github.com/VATSIM-UK/core/pull/1461/files#diff-d4c8c6dc8769324fc27cfdac19f05cafR50). It might just be a case of doing the petty renaming of stuff when we come to merge nova into master, but up to you!

Reply:
Yeah, I'd rather do that when we get the feature/epic done if that's alright?